packages/contracts/test/rbac.ts

```diff
@@ -0,0 +1,390 @@
+import { Provider, ProviderRegistry } from "@blockstack/clarity";
+import { expect } from "chai";
+import { resolve } from "dns";
+import { RBACClient } from "../src/rbac-client"
+import { Clarity } from "../src/util"
+
+
+
+describe("rbac contract test suite", () => {
+  let provider: Provider;
+  let client: RBACClient;
+  const DEPLOYER = 'SP3GWX3NE58KXHESRYE4DYQ1S31PQJTCRXB3PE9SB';
+  const USER = 'STB44HYPYAT2BB2QE513NSP81HTMYWBJP02HPGK6';
+
+
+  describe("Contract syntax", async () => {
+    it("should be valid", async () => {
+      provider = await ProviderRegistry.createProvider();
+      client = new RBACClient(DEPLOYER, provider);
+
+      await client.checkContract();
+      await client.deployContract();
+    });
+
+    after(async () => {
+      await provider.close();
+    });
+  });
+
+  describe("Contract logic [Roles]", async () => {
+    beforeEach(async () => {
+      provider = await ProviderRegistry.createProvider();
+      client = new RBACClient(DEPLOYER, provider);
+
+      await client.deployContract();
+    });
+
+   
+    afterEach(async () => {
+      await provider.close();
+    });
+
+
+    it("returns false user have no roles at all", async () => {
+      const receipt = await client.hasRole(USER, 123);
+
+      expect(Clarity.unwrapBool(receipt)).false;
+    });
+
+   
+    it("should fail granting role higher than 127", async () => {
+      for (let i = 128; i <=150; i++) {
```

Comment:
It's a random value chosen by my tired brain.
In theory we could get rid of this loop and run test only against value 128, but I decided to test few more values just in case.